helm/defectdojo/values.yaml

```diff
@@ -281,6 +281,7 @@ django:
 
 initializer:
   run: true
+  metadataAnnotations: {}
```

Comment:
What is the difference between annotations and metadataAnnotations?

Reply:
ok changed it to jobAnnotations